_shared/project/.github/workflows/ci.yml

```diff
@@ -99,5 +98,5 @@ jobs:
       run: make frontend-lint
     - name: Test
       run: make frontend-test
-  {%- endif %}
-  {{- include(".github/workflows/ci.yml", indent=2) }}
+  {% endif %}
+  {{- include(".github/workflows/ci.yml", indent=2) -}}
```

Comment:
I'm surprised this doesn't cause trouble in YAML files, but it looks like we were fighting a lot of whitespace stuff here anyway